<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QDialog" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>221</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QDialogButtonBox" name="okCancel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>180</y>
     <width>181</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="portText">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid gray;
border-radius: 8px;
padding-left: 2px;
padding-right: 2px;</string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
   <property name="text">
    <string>8001</string>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="portLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Port:</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="serviceBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Service</string>
   </property>
   <widget class="QRadioButton" name="clientButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>117</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="text">
     <string>Client</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="serverButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>117</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Server</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="clientBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>201</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Client Settings</string>
   </property>
   <widget class="QLabel" name="ipLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>IP:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ipText">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid gray;
border-radius: 8px;
padding-left: 2px;
padding-right: 2px;</string>
    </property>
   </widget>
   <widget class="QLabel" name="aliasLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Alias:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="aliasText">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>46</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid gray;
border-radius: 8px;
padding-left: 2px;
padding-right: 2px;</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>0</y>
     <width>261</width>
     <height>221</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: url(./ostrichSettingsBG.png);</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>portText</tabstop>
  <tabstop>ipText</tabstop>
  <tabstop>clientButton</tabstop>
  <tabstop>serverButton</tabstop>
  <tabstop>okCancel</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>okCancel</sender>
   <signal>accepted()</signal>
   <receiver>Settings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>okCancel</sender>
   <signal>rejected()</signal>
   <receiver>Settings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
